### ISSUE TITLE
UI for VCP alternates

### ISSUE THREAD

**Issue by funderburkjim** [sanskrit-lexicon/alternateheadwords#17]
This issue will be devoted to developing a UI for examining questionable cases
of alternate headwords of VCP.

**Comment by funderburkjim** [comment]
@gasyoun  Would something like the [AP corrections UI](http://www.sanskrit-lexicon.uni-koeln.de/scans/APScan/2014/pywork/correctionwork/issue-334/208/update.php?input=deva) be what we need?

### Left-hand section list of cases
* Maybe `key1 (key1alt)` 

### Main section consisting of:
* a form for correcting or confirming key1alt   (maybe allow also correction of key1, which might rarely
   need changing)
   *  include comment field
   * change/no-change radio buttons
   *  maybe skip the typo/print distinction, as less important and a potential source of confusion to 
Radha
   * submit and cancel buttons
* Supporting data
   * A display of first few lines of the VCP record for the headword of the case
   * A Link to the scan for this headword.
     *  Probably I can make the 'green blob' type of link to approximate position of first line of
          headword in scan page
   * links to headwords in other dictionaries (for the key1).

Do you think Radha prefers to work in Devanagari? If not, what?

**Comment by funderburkjim** [comment]
@gasyoun   Do you have any comments?  If you think the UI suggestion is a good approach for Radha, then I'll proceed with developing it.

**Comment by gasyoun** [comment]
@funderburkjim 

Legend.
Green - untouched?
Green italics - ?
Orange - ?

It's a bit too overloaded, are you able to tidy up a bit?

![apap](https://cloud.githubusercontent.com/assets/80761/24892042/74fa814e-1e82-11e7-9e94-27d2b5efcbf8.PNG)

**Comment by funderburkjim** [comment]
So you concur with adapting the basic plan of the display to the current VCP situation,
but  taking into account the changes of your annotation?

**Comment by gasyoun** [comment]
> So you concur with adapting the basic plan of the display to the current VCP situation,
but taking into account the changes of your annotation?

Did not get the message.

**Comment by funderburkjim** [comment]
> Did not get the message.

Rephrase:  Is the above display a good one to for me to work on for Radha?  If you concur, I'd like to
get started with it.

**Comment by gasyoun** [comment]
> Rephrase: Is the above display a good one to for me to work on for Radha?

If minor changes can be done (like not fitting inside the box) can be fixed, great. If not - not a big issue. It's good to go.

**Comment by funderburkjim** [comment]
[Here](http://www.sanskrit-lexicon.uni-koeln.de/scans/VCPScan/2013/pywork/correctionwork/ahw-issue-017/update.php?batch=test1)  is a version of the UI.

It is a test batch containing the first 10 cases where the alternate headword needs to be confirmed or altered.

@gasyoun   Play around a bit with it and see if you think it is good to go for Radha.

When you confirm, I'll prepare batches  of 25 cases  (there will be 8 or 9 such batches)  and will make
a 'progress' table (such as [this](https://github.com/sanskrit-lexicon/CORRECTIONS/issues/336#issuecomment-277861728) , and a table of links.

The UI currently works in slp1 or devanagari (with extra parameter in url `&input=deva` ).  I wasn't sure how Radha would want to make the changes and view the data.

**Comment by funderburkjim** [comment]
@gasyoun   Did you get Radha started on this yet?

**Comment by gasyoun** [comment]
@funderburkjim no, I can't make a video on it. I mean to make a video I need an algorithm, but here myself I'm not sure how it should be done. She is ready, but not I. That's why I'm sad, she is not doing, but still ready to help.

**Comment by funderburkjim** [comment]
Should I try to make a video for you, to help you understand algorithm?
